lib/revision_table_access_check_filter.js

```diff
@@ -81,6 +81,13 @@ module.exports = function(hyper, req, next, options, specInfo) {
                 }
             });
         }
-        return next(hyper, req);
+        return next(hyper, req)
+        .then(function(res) {
+            if (req.query.redirect === false || mwUtil.isNoCacheRequest(req)) {
+                res.status = 200;
+                delete res.headers.location;
```

Comment:
Right, but even with `redirect=false` the SLA is to return the location header. And actually Varnish counts on that, it just changes the return code based on the `redirect` flag, so this represents a breaking change. It should be harmless leaving it in now that the location is URI-encoded, shouldn't it?